docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,26 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |add details to a certain contact after adding them |have more information available afterwards
+
+|`* * *` |user |be notified if I were to delete a contact |I don’t mistakenly delete a contact
+
+|`* * *` |user |have a more flexible addition of contact |add contacts without address or email as compulsory elements
+
+|`* * *` |user |show list of favourite contacts |quickly find people I contact often
+
+|`* * *` |user |be notified when entering user with same name |avoid confusion in the future
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |use link:#right-click-menu[right click menu] for most link:#common-commands[common commands] |no typing is required and makes my life easy
+
+|`* *` |user |show contact according to categories of labels |access certain group by label
+
+|`* *` |user |copy email address to clipboard |paste it into my email application’s recipients field
+
+|`* *` |user |have wrong command to be highlighted |change it immediately
+
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
```

Comment:
Overall, the user stories follow the correct form. However, some are not user stories, instead they should be in use case or NFR. Please update them accordingly. Also, don't forget to define the target user.